Corgibytes.Freshli.Cli/Functionality/Git/IGitManager.cs

```diff
@@ -5,4 +5,6 @@ public interface IGitManager
     public GitCommitIdentifier ParseCommitId(string commitId);
 
     string CreateArchive(string repositoryId, GitCommitIdentifier gitCommitIdentifier);
+
+    public bool GitRepositoryInitialized(string repositoryLocation, IConfiguration configuration);
```

Comment:
I think this name would better indicate that this is a predicate method:

```suggestion
    public bool IsGitRepositoryInitialized(string repositoryLocation, IConfiguration configuration);
```